src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,74 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int TRANSACTION_TYPE_POSITION = 0;
+    private static final int MONEY_AMOUNT_POSITION = 1;
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
```

Comment:
I dont think we need "supply" and "buy" as constnats

Reply:
How to determine where we need create constant and where not?